src/tests/ftest/daos_test/dfuse.py

```diff
@@ -100,6 +100,17 @@ def run_test(self, il_lib=None):
                 daos_test_env['D_IL_MAX_EQ'] = '2'
                 daos_test_env['D_IL_ENFORCE_EXEC_ENV'] = '1'
 
+        command = [
```

Comment:
Looks like you have a merge conflict here.  This new alignment is nice and will reduce conflicts in future but you do need to remove L114-115.

Reply:
> Arghhh, merging failure from my side.
> 
> * [x]  Remove deprecated command

Fixed with commit dd12a3828